test/plugins/surround.test.ts

```diff
@@ -32,6 +33,20 @@ suite('surround plugin', () => {
     end: ['first |(line) test'],
   });
 
+  newTest({
```

Comment:
`ysw` doesn't work in some cases. For instance, put the cursor at the start of `newTest` here and do `ysw"`

Reply:
Seems like this happens if there's a newline or punctuation before whitespace

Reply:
See https://github.com/VSCodeVim/Vim/pull/6452/commits/20629ff7fd36893454179ce27d737b278bfbd7c6 with whole function as context.
Apparently, we are expected to correct w/aw motions, which is a bit ugly.
I thought about/would prefer adding maps: ysw -> yse but that would not work with counts: ys4w.

This seems to handle all known cases correct.